### PR TITLE
feat: adding ssri.create for a crypto style interface

### PR DESCRIPTION
**Pull request by soldair**
adding crypto.create
adding toJSON to Integrity

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -275,6 +281,43 @@ function integrityStream (opts) {
   return stream
 }
 
+module.exports.create = createIntegrity
+function createIntegrity (opts) {
+  opts = opts || {}
+
+  opts = opts || {}
```
Comment (zkat) [inline index.js]: doubleplus options!

File: README.md [hunk 2]
```diff
@@ -200,6 +202,22 @@ const integrity = 'sha512-9KhgCRIx/AmzC8xqYJTZRrnO8OW2Pxyl2DIMZSBOr0oDvtEFyht3xp
 ssri.parse(integrity).toString() === integrity
 ```
 
+#### <a name="integrity-to-json"></a> `> Integrity#toJSON() -> String`
+
+Returns the string representation of an `Integrity` object. All hash entries
+will be concatenated in the string by `' '`.
+
+this is a convenience method so you can pass an `Integrity` object directly to `JSON.stringify`
```
Comment (zkat) [inline README.md]: Minor nit: capitalization

File: README.md [hunk 3]
```diff
@@ -200,6 +202,22 @@ const integrity = 'sha512-9KhgCRIx/AmzC8xqYJTZRrnO8OW2Pxyl2DIMZSBOr0oDvtEFyht3xp
...
+this works with the built in native JSON feature. For more info check out [toJSON() behavior on mdn](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify#toJSON%28%29_behavior)
```
Comment (zkat) [inline README.md]: Minor nit: period at end of sentence?

File: README.md [hunk 4]
```diff
@@ -312,6 +330,26 @@ ssri.fromStream(fs.createReadStream('index.js'), {
 }) // succeeds
 ```
 
+#### <a name="create"></a> `> ssri.create([opts]) -> <Integrity>`
+
+Returns an Integrity object calculated by reading data from
+calls to update
```
Comment (zkat) [inline README.md]: Minor nit: period missing.

**Review by zkat — APPROVED** [review]
Thanks a bunch for this PR! This is a really handy mechanism that I'm glad we're gonna have available <3

I added a few pretty minor nitpicks. Up to you whether you want to fix them, but it would be nice. :)

**Review by zkat — APPROVED** [review]

**Comment by soldair** [comment]
=) feel free to merge when you are ready

**Pull request merged**